### PR TITLE
Remove explicit mention of `[heketi]` string in error message

### PR DESCRIPTION
**Pull request by humblec**
Signed-off-by: hchiramm <hchiramm@redhat.com>

### CODE REVIEW

File: gluster/block/cmd/glusterblock-provisioner/glusterblock-provisioner.go [hunk 1]
```diff
@@ -49,7 +49,7 @@ const (
 	creatorAnn         = "kubernetes.io/createdby"
 	volumeTypeAnn      = "gluster.org/type"
 	descAnn            = "Gluster-external: Dynamically provisioned PV"
-	provisionerVersion = "v2.0.0"
+	provisionerVersion = "v5.0.0"
```
Comment (wongma7) [inline gluster/block/cmd/glusterblock-provisioner/glusterblock-provisioner.go]: IDK what version it's supposed to be at but okay!

**Comment by humblec** [comment]
@wongma7 not sure how can approve this PR :)

**Pull request merged**